plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -596,4 +632,44 @@ public void afterRunning(Runnable runnable)
                     .isEqualTo(expectedMisses);
         }
     }
+
+    private static final class TestingJdbcIdentityCacheMapping
+            implements JdbcIdentityCacheMapping
```

Comment:
After we move to Java 15, it would be great to seal the `JdbcIdentityCacheMapping` class.
Would it be possible to write this test in a way that wouldn't require adding a test implementation?